README.md

```diff
@@ -25,10 +25,15 @@ npm run test
 npm run test:unit
 ```
 
-To run the application in dev mode with a nodemon watcher
-
+To install npm CLI tool. It will be linked to local binaries so can be executed as `process-management`
 ```shell
-npm run dev
+npm i -g
+
+# using CLI
+process-management help
+
+# example
+process-management create -p 20 -h paulius.org '[{}]'
 ```
 
 ## Library functions
```

Comment:
the restrictions format in this section is out of date, still talks about using the old `RestrictionName: [RestrictionValue]` structure

Reply:
This is not a complete example, there is one in `process-management help create`

Reply:
I'm talking about `Restrictions are provided as JSON in the format `{ RestrictionName: [RestrictionValue] } `. For example:` onwards, can probably just delete it as we have the CLI explanation now